app/ui/src/main/java/com/fsck/k9/fragment/MessageListFragment.java

```diff
@@ -2526,10 +2526,12 @@ public void onLoadFinished(Loader<Cursor> loader, Cursor data) {
         if (cursors.length > 1) {
             cursor = new MergeCursorWithUniqueId(cursors, getComparator());
             uniqueIdColumn = cursor.getColumnIndex("_id");
+
         } else {
             cursor = data;
             uniqueIdColumn = ID_COLUMN;
         }
+        if (adapter != null) adapter.setUniqueIdColumn(uniqueIdColumn);
```

Comment:
A method on `adapter` is called without a `null` check further down. So this one shouldn't be necessary either.

Reply:
done